### DISCUSSION TITLE
【keyball44】 LED9番の基盤破損

### DISCUSSION THREAD

**Discussion by Pary1217** [Yowkees/keyball#675] [Q&A]
keyball44 にてLED9番の基盤を破損してしましました。
ジャンパ線にて修復したいのですが9番は裏向きのLEDなのであきらめて
8番→10番と配線しようと考えておりますが
8番2DOUT→10番4DINに接続するだけでいけますでしょうか。

![20241011_234251](https://github.com/user-attachments/assets/44215761-0f69-4116-aaad-327850a1e093)
![20241011_234251](https://github.com/user-attachments/assets/d1a0edd8-e835-4d41-add2-82e4213ece76)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。返答が遅くなり申し訳ありませんでした。

おっしゃる通り、青線のように接続していただければ、９番をスキップして１０番以降も点灯します。
お手数ですがご対応お願いいたします。
